tools/interop_matrix/client_matrix.py

```diff
@@ -64,14 +64,14 @@ def get_github_repo(lang):
         'v1.6.1',
         'v1.7.0',
     ],
-    #'python': [
-        #'v1.0.x',  #Fail to run the test. #13230.
-    #    'v1.1.4',
-    #    'v1.2.5',
-    #    'v1.3.9',
-    #    'v1.4.2',
-    #    'v1.6.6',
-    #],
+    'python': [
+        'v1.0.x',  #Fail to run the test. #13230.
+        'v1.1.4',
+        'v1.2.5',
+        'v1.3.9',
+        'v1.4.2',
+        'v1.6.6',
```

Comment:
Any desire to add a `v1.7.x` tag to this list?

Reply:
It'll be in #13398.